enetwork

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+## Variables, fixed for now, will be env vars or secrets later
+GatewayUsername=${SEMAPHORE_DOCKERGW_USER,-''}
```

Comment:
What does `${SEMAPHORE_DOCKERGW_USER,-''}` do?

Reply:
Checks if its set, if not defaults to ''